packages/cmd/src/help/DefaultHelpGenerator.ts

```diff
@@ -210,13 +211,15 @@ export class DefaultHelpGenerator extends AbstractHelpGenerator {
             let maximumLeftHandSide = 0;
             for (const command of definitions) {
                 let summaryText: string = "";
+                summaryText += command.summary || command.description;
 
-                // Mark with the experimental tag if necessary
-                if (command.experimental) {
+                if (command.deprecatedReplacement) {
+                    // Mark with the deprecated tag
+                    summaryText += this.grey(" (deprecated)");
+                } else if (command.experimental) {
+                    // Mark with the experimental tag
                     summaryText += this.grey("(experimental) ");
```

Comment:
Should this be changed to `" (experimental)"` since it seems to be a suffix now rather than a prefix? (related to above comment)

Reply:
Yes a space should come before (experimental).